### PR TITLE
Connect to notion db to update containment measures

### LINKED ISSUE #106: Automatically and regularly load data from containment measures database
@Bachibouzouk made [a PR](https://github.com/epidemics/covid/pull/88) for displaying containment measures from a .csv exported from [the Notion database of containment measures] on country pages

We ideally want this feeding of data to be automatic rather than manual; and to have it update whenever the Notion database updates. 

Notion doesn't have an API, but there's [an unofficial Python API wrapper](https://medium.com/@jamiealexandre/introducing-notion-py-an-unofficial-python-api-wrapper-for-notion-so-603700f92369)

This issue requires further speccing out for what's needed. This is just a reminder that it needs doing.

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Fix #106 

Automatically and regularly load data from containment measures database

- [x] add `notion` to the dependencies of `poetry`

- [x] get the token from  `NOTION_TOKEN` in github secrets for the docker deploy (related to #173)

@hnykda -told me he would take care of these two points after a night of sleep

To make it work after the merge:

 -   enableNotion in values.yaml set to True
 -   set up the token in the secrets.yaml to the one which should be used by the app (encrypted - I can give you permissions or encrypt it for you). Note that we need some token which is persistent (which I am not sure the webbrowser is).

### CODE REVIEW

File: src/server/notion_connect.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+from notion.client import NotionClient
+
+# TODO use github secrets
+# Obtain the `token_v2` value by inspecting your browser cookies on a logged-in session on Notion.so
+token_v2 = "NOTION_TOKEN on github secrets"
```
Comment (Bachibouzouk) [inline src/server/notion_connect.py]: @hnykda - here is the place where I need to token

**Comment by hnykda** [comment]
I did the secret part, should work fine now (tested on staging), see the related PR #182

**Comment by jk3141** [comment]
The measures should be now listed under the "predictions" graph, that means not in the sidebar.

**Comment by jk3141** [comment]
https://github.com/epidemics/covid/issues/191

**Comment by Bachibouzouk** [comment]
> The measures should be now listed under the "predictions" graph, that means not in the sidebar.

Ok, so I will move them below to fix #191 then, or will that clash with someone elses PR?

**Comment by hnykda** [comment]
I would merge this as a BE part and worked on FE separately.

**Comment by Bachibouzouk** [comment]
> I would merge this as a BE part and worked on FE separately.

ok

**Pull request merged**